src/components/event-card/event-card.component.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactElement } from "react";
+import Image from "next/image";
+
+import HugeiconsCalendar03 from "@/icons/HugeiconsCalendar03";
+import HugeiconsLocation04 from "@/icons/HugeiconsLocation04";
+import HugeiconsShoppingBag02 from "@/icons/HugeiconsShoppingBag02";
+
+import { AvailableEvent } from "@/lib/data";
+
+import styles from "./event-card.module.css";
+
+type Props = {
+  eventsData: AvailableEvent;
+};
+
+export default function EventCardComponent({
+  eventsData,
+}: Props): ReactElement {
+  return (
+    <div className={styles["event-card"]}>
+      <div className={styles.visuals}>
+        {/* add sold out image here */}
```

Comment:
یه todo اولش بذارید که بعداً بتونید راحت پیداش کنید

```suggestion
        {/* todo: add sold out image here */}
```